src/njoy21/input/ACER/Card10/Tempd/test/Tempd.test.cpp

```diff
@@ -0,0 +1,36 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "Tempd input values", "[ACER], [Card10], [Tempd]" ){
+  GIVEN( "valid tempd values" ){
+    std::vector<double> validTempd{ 0.0, 293.6 };
+
+    THEN( "the returned class has the correct value" ){
+      for( auto tempd : validTempd ){
+	iRecordStream<char> issTempd( std::istringstream( std::to_string(tempd) ) );
+        auto tempdArg = argument::extract< ACER::Card10::Tempd >(issTempd );
+        REQUIRE( tempd * dimwits::kelvin == tempdArg.value );
+      }
+    }
+  }
+  GIVEN( "no tempd value" ){
+    THEN( "the default value is returned" ){
+      iRecordStream<char> iss( std::istringstream( "/") );
+      REQUIRE( ACER::Card10::Tempd::defaultValue() == 
```

Comment:
as written, this test is redundant with the argument parser tests. If you're going to testing default values, specify the value in the test such that if it's changed in the future, we know.